olive/passes/onnx/transformer_optimization.py

```diff
@@ -140,6 +141,26 @@ def _run_for_config(
         for key in get_external_data_config():
             del run_config[key]
 
+        if model.hf_config:
+            model_config = model.hf_config.load_model_config().to_dict()
+            input_model_type = model_config.get("model_type", "")
+            _model_type = MODEL_TYPE_MAPPING.get(input_model_type, input_model_type)
+            assert _model_type in transformers_optimizer.MODEL_TYPES, (
```

Comment:
This assert might have to be moved after line 152. Otherwise, this assert will fail even though the user provide the value through `run_config["model_type"]`

Reply:
Thx! Updated.